integration/spring-boot/autoconfigure/src/main/java/org/apache/cxf/spring/boot/autoconfigure/ssl/CxfClientSslProperties.java

```diff
@@ -0,0 +1,109 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.cxf.spring.boot.autoconfigure.ssl;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * Client-side SSL integration properties for CXF when Spring Boot SslBundles are present.
+ * bundles are applied once when an HTTPConduit is configured.
+ */
+@ConfigurationProperties("cxf.client.ssl")
+public class CxfClientSslProperties {
```

Comment:
Not particularly better option, but may be we could mirror SSL bundle with list of named `CxfClientSslBundle` instances? Fe:

```
@ConfigurationProperties("cxf.client.ssl")
public class CxfClientSslProperties {
    List<CxfClientSslBundle> bundles;
    private boolean enabled;

    class CxfClientSslBundle {
        private String name; <--- same as the name in `HTTPConduitConfigurer`
        private Boolean disableCnCheck = Boolean.FALSE;
        private String bundle;
        private String protocol;
        private List<String> cipherSuites;
    }
}
```

To me the combination of props + rules does make it confusing somewhat